tests/manual/tableselection.html

```diff
@@ -0,0 +1,57 @@
+<style>
+	body {
+		font-family: Helvetica, Arial, sans-serif;
+		font-size: 14px;
+	}
+
+	.print-selected {
+		background: #b8e3ff;
+	}
+</style>
+
+<div id="editor">
+	<p>A table to test selection:</p>
```

Comment:
This test's content is too homogenous to properly test the feature (as follow-ups has shown). We need more cases, for instance:
* with the content before and after the table,
* with a solitary table in the content,
* with a table that has some cells already split/merged,
* with a table that has links, block quotes and other content in some cells.

Reply:
I have something already for some of the points in the copy table fragment PR. I can cherry pick it here.

Reply:
But I'm not sure how to handle all these issues in one manual test. I've added only a complex table (which covers two of those issues).

Reply:
Create 2 editors? 😛

Reply:
But we have 4 or more cases :P 

But what do you think about loading fixtures on a button? One editor (I'd like to have one editor as copy-paste manual tests enhancement would require an editiable arae side by side to the editor.

Reply:
Moved to: https://github.com/ckeditor/ckeditor5/issues/6306